controllers/api/webhook.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+//TODO
+
+import (
+  // "encoding/json"
+  // "errors"
+  "net/http"
+
+  "github.com/gophish/gophish/models"
+  // "github.com/gorilla/mux"
+)
+
+
+
+func (as *Server) Webhooks(w http.ResponseWriter, r *http.Request) {
+  switch {
+  case r.Method == "GET":
+    wh_s, err := models.GetWebhooks()
```

Comment:
Minor nit - I try not to use `_`'s in variable names where possible. In this case, `whs` would be fine.